src/components/course/RecordingPlacement.vue

```diff
@@ -22,6 +22,9 @@
           </div>
         </div>
         <div v-if="isEditing">
+          <label for="select-publish-type" class="sr-only">
+            Publish options
+          </label>
```

Comment:
@johncrossman I think this added label is going to result in the same problem. The `label` attribute on the <v-radio> causes it to render a <label>.